react/src/pages/GacComparePage.js

```diff
@@ -0,0 +1,256 @@
+                                )}
+                            </select>&nbsp;<span style={{color:"blue"}}>&#x2193;</span>
+                            <br />
+                            {getGacName(response.get("gac_compare"))}
+                        </td>
+                    </tr>
+                    <tr><td style={{height:"3px"}} colSpan="4"></td></tr>
+                    <tr><td style={{height:"1px",background:"darkblue"}} colSpan="4"></td></tr>
+                    <tr><td style={{height:"8px"}} colSpan="4"></td></tr>
+                    </thead>
+                    <tbody style={{fontSize:"10pt"}}>
+                    { unique_keys?.map((key, keyIndex) => {
+                        return <React.Fragment key={Uuid()}>
+                            { (showingAll() ||
+                               (showingMatches() && sameGacValue(key, response.data)) ||
+                               (showingNonMatches() && !sameGacValue(key, response.data)) ||
+                               (showingDifferences() && !sameGacValue(key, response.data) && !(addedGacValue(key, response.data) || removedGacValue(key, response.data))) ||
+                               (showingMissing() && (addedGacValue(key, response.data) || removedGacValue(key, response.data)))) ?
+                            (<React.Fragment>
+                            <tr key={Uuid()} style={{color:sameGacValue(key, response.data) ? "inherit" : (removedGacValue(key, response.data) || addedGacValue(key, response.data) ? "red" : "darkred")}}>
+                            <td>
+                                {sameGacValue(key, response.data) ? <span>&#x2713;</span> : <span>&#x2717;</span>}&nbsp;
+                            </td>
+                            <td>
+                                <b>{key}</b>
+                            </td>
+                            <td style={{paddingRight:"8pt"}}>
+                                <span>{addedGacValue(key, response.data) ? <b>MISSING</b> : ((appearsToBeObfuscated(response.data.gac[key]) ? <b>OBFUSCATED</b> : response.data.gac[key]) || <b>&#x2205;</b>)}</span>
+                            </td>
+                            <td>
+                                <span>{removedGacValue(key, response.data) ? <b>MISSING</b> : ((appearsToBeObfuscated(response.data.gac_compare[key]) ? <b>OBFUSCATED</b> : response.data.gac_compare[key]) || <b>&#x2205;</b>)}</span>
+                            </td>
+                            </tr>
+                            { keyIndex < unique_keys.length - 1 ? (<React.Fragment>
+                                <tr><td style={{height:"4px"}}></td></tr>
+                                <tr><td style={{height:"1px",background:"lightblue"}} colSpan="4"></td></tr>
+                                <tr><td style={{height:"3px"}}></td></tr>
+                             </React.Fragment>):(<tr/>)}
```

Comment:
I think it's allowed; React doesn't complain about it (on the console output), and it does seem to complain about table related element inconsistencies quite religiously.

Reply:
The notation is probably allowed in XHTML but I didn't even think it existed at all as a notation in HTML. Anyway I guess probably not a huge deal. since HTML will ignore tokens like that. I often do `<BR />` in HTML I want to be portable but I do a separator so that old tokenizers don't have trouble and just see it as an ignorable attribute.